src/apps/competitions/tests/test_submissions.py

```diff
@@ -0,0 +1,121 @@
+from datetime import timedelta
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from competitions.models import Competition, Phase, Submission
+from datasets.models import Data
+from django.utils.timezone import now
+
+User = get_user_model()
+
+
+class SubmissionsTests(TestCase):
+    def setUp(self):
+        self.user = User.objects.create(
+            username='tester'
+        )
+        self.user.set_password('testing')
+        self.user.save()
+        self.competition = Competition.objects.create(
+            title='Test Competition',
+            created_by=self.user,
+        )
+        self.phase = Phase.objects.create(
+            competition=self.competition,
+            index=0,
+            start=now(),
+            name='Test Phase',
+        )
+
+    def make_submission(self, user=None, phase=None, data=None, submission_status=None, submission_date=None, submission_name=None):
```

Comment:
NOTE: Should move this somewhere common and/or use factory boy, when you implement this issue #91